src/odemis/gui/cont/settings.py

```diff
@@ -558,35 +559,57 @@ def _create_controls(self):
         self._pnl_acqfile = FileInfoSettingsController(self.tab_panel.fp_fileinfo, "No file loaded")
         wildcards, _ = formats_to_wildcards(odemis.dataio.get_available_formats(),
                                                             include_all=True)
-        # Panel with AR background file information
+
+        # Panel allowing to load bg correction or calibration files
+        # Settings displayed are stream specific
+        self._pnl_calibration = FileInfoSettingsController(self.tab_panel.fp_fileinfo, "")
+
+        # Display with AR background file information
         # It's displayed only if there are AR streams (handled by the tab cont)
-        self._pnl_arfile = FileInfoSettingsController(self.tab_panel.fp_fileinfo, "")
-        self._arfile_ctrl = self._pnl_arfile.add_file_button(
+        self._arfile_ctrl = self._pnl_calibration.add_file_button(
             "AR background",
             tooltip="Angle-resolved background acquisition file",
             clearlabel="None", wildcard=wildcards).value_ctrl
-        self._pnl_arfile.hide_panel()
+        self._ar_bckfile_entry = self._pnl_calibration.entries[0]
+        self._ar_bckfile_entry.lbl_ctrl.Show(False)
+        self._ar_bckfile_entry.value_ctrl.Show(False)
```

Comment:
The function `add_file_button()` returns the entry. So no need to guess try to find it again via the `_pnl_calibration.entries`. You should be able to get rid of `._arfile_ctrl`, and replace it everywhere by `._ar_bckfile_entry.value_ctrl`.

(Same thing for the next three file buttons).